.github/scripts/install_test_binaries.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+# Installs Solc and Geth binaries
+# Note: intended for use only with CI (x86_64 Ubuntu, MacOS or Windows)
+set -e
+
+GETH_BUILD=${GETH_BUILD:-"1.11.2-73b01f40"}
+
+BIN_DIR=${BIN_DIR:-"$HOME/bin"}
+
+PLATFORM="$(uname -s | awk '{print tolower($0)}')"
+if [ "$PLATFORM" != "linux" ] && [ "$PLATFORM" != "darwin" ]; then
+    EXT=".exe"
+fi
+
+main() {
+    mkdir -p "$BIN_DIR"
+    cd "$BIN_DIR"
+    export PATH="$BIN_DIR:$PATH"
+    if [ "$GITHUB_PATH" ]; then
+        echo "$BIN_DIR" >> "$GITHUB_PATH"
+    fi
+
+    install_geth &
+    g=$!
+    install_solc &
+    wait $g $!
+
+    echo ""
+    echo "Installed Geth:"
+    geth version
+    echo ""
+    echo "Installed Solc:"
+    solc --version
+}
+
+# Installs geth from https://geth.ethereum.org/downloads
+install_geth() {
+    case "$PLATFORM" in
```

Comment:
is it worth caching and then shortcutting if tool is already installed?

Reply:
hmm, would need to check version in case they differ, but we're only downloading binary so it's not that important imo